include/z64.h

```diff
@@ -802,9 +802,9 @@ typedef struct {
 typedef struct {
     /* 0x0 */ u32 useRgba;
     /* 0x4 */ u32 setScissor;
-    /* 0x8 */ Color_RGBA8 primColor;
-    /* 0xC */ Color_RGBA8 envColor;
-} struct_801F8020; // size = 0x10
+    /* 0x8 */ Color_RGBA8_u32 primColor;
```

Comment:
The problem here is that it has to use the union, because the bytes are set individually in Init but later used as a single u32 in the Draw, which is what this type is intended to be for.